packages/template-retail-react-app/app/components/image-gallery/index.jsx

```diff
@@ -72,7 +72,9 @@ const ImageGallery = ({imageGroups = [], selectedVariationAttributes = {}, size}
     const heroImageGroup = useMemo(
         () =>
             findImageGroupBy(imageGroups, {
-                viewType: LARGE,
+                // @@@ If only `hi-res` images are avaiable, use those.
+                // viewType: LARGE,
+                viewType: 'hi-res',
```

Comment:
Different catalogs use different image groups. We need to make the image group settings used across components customizable.